project/ws/app/src/lib/routes/profile-v2/routes/profile-view/profile-view.component.html

```diff
@@ -392,81 +764,6 @@
-          <div [attr.id]="discussions"
-            class="flex flex-1 flex-column margin-top-xs margin-botton-xs dark-card discussion-card"
-            style="box-shadow: none"> -->
-        <!-- <mat-card [attr.id]='discussions' class="margin-left-xs margin-top-xs margin-botton-xs dark-card discussion-card"
-          style="box-shadow: none;"> -->
-
-        <!-- <mat-card-content> -->
-        <!-- <div class="flex flex-1">
-              <span class="margin-bottom-xs margin-top-l mat-title flex-1">{{
-                'Discussions'
-                }}</span> -->
-        <!-- <div
-                class="ml-4 mr-6 mb-2 view-more ws-mat-accent-text view-all ws-mat-primary-lite-background-important">
-                <a mat-button [routerLink]="'/app/discuss/my-discussions'">
-                   <span i18n>{{'Go to discussion profile'}}</span>
-                </a>
-              </div> -->
-        <!-- </div>
-            <ng-container> -->
-        <!-- <mat-card class="recent-tabs"> -->
-        <!-- <div class="w-full">
-                <div class="flex flex-1 flex-column filter_outer_box">
-                  <div class="flex flex-1 filter flex-wrap">
-                    <a href="javascript:void(0)" class="filter-option font-normal" i18n-aria-label role="button"
-                      [ngClass]="{
-                        'ws-mat-accent-border-active':
-                          currentFilter === 'timestamp',
-                        'ws-mat-accent-border font-medium':
-                          currentFilter !== 'timestamp'
-                      }" mat-button (click)="filter('timestamp')">
-                      {{ 'profileView.recentPosts' | translate }}
-                    </a>
-                    <a href="javascript:void(0)" class="filter-option" i18n-aria-label role="button" [ngClass]="{
-                        'ws-mat-accent-border-active': currentFilter === 'best',
-                        'ws-mat-accent-border font-medium':
-                          currentFilter !== 'best'
-                      }" mat-button (click)="filter('best')">
-                      {{ 'profileView.bestPosts' | translate }}
-                    </a>
-                    <a href="javascript:void(0)" class="filter-option" i18n-aria-label role="button" [ngClass]="{
-                        'ws-mat-accent-border-active':
-                          currentFilter === 'saved',
-                        'ws-mat-accent-border font-medium':
-                          currentFilter !== 'saved'
-                      }" mat-button (click)="filter('saved')" *ngIf="
-                        portalProfile.id === currentUser?.userId ||
-                        portalProfile.userId === currentUser?.userId
-                      ">
-                      {{ 'profileView.savedPosts' | translate }}
-                    </a>
-                  </div>
-                </div> -->
-        <!-- </mat-card> -->
-        <!-- </div>
```

Comment:
please remove unwanted code, if not required

Reply:
Will remove once I implement all the functionality, as I need this code as backup or for reference